flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/JobManagerConfigHandler.java

```diff
@@ -40,12 +41,22 @@ public JobManagerConfigHandler(Configuration config) {
 	public String handleRequest(Map<String, String> pathParams, Map<String, String> queryParams, ActorGateway jobManager) throws Exception {
 		StringWriter writer = new StringWriter();
 		JsonGenerator gen = JsonFactory.jacksonFactory.createGenerator(writer);
-
 		gen.writeStartArray();
+
 		for (String key : config.keySet()) {
 			gen.writeStartObject();
 			gen.writeStringField("key", key);
-			gen.writeStringField("value", config.getString(key, null));
+			if(key.equals(ConfigConstants.SECURITY_COOKIE)) {
+				String value = config.getString(key, null);
+				if(value != null) {
+					value = "******";
+				}
+				gen.writeStringField("value", value);
+			}
+			else {
+				gen.writeStringField("value", config.getString(key, null));
+			}
```

Comment:
Good idea not to expose the setting here!
The `GlobalConfiguration.loadYAMLResource()` method is logging all the key / value pairs. Maybe it makes sense to hide the value there as well.